.travis.yml

```diff
@@ -8,18 +8,14 @@ jobs:
       env: NAME="clang-format"
       addons:
         apt:
+          sources:
+            - llvm-toolchain-trusty-6.0
           packages:
-            - clang-format-3.8
+            - clang-format-6.0
       install:
       script: |
-        # Apparently update-alternatives doesn't work in Travis containers
-        mkdir -p priority-symlinks
-        ln -s /usr/bin/clang-format-3.8 priority-symlinks/clang-format
-        export PATH=${PWD}/priority-symlinks:${PATH}
-
-        # Now we can do the formatting pass
         clang-format --version
-        git-clang-format-3.8 "${TRAVIS_BRANCH}"
+        git-clang-format-6.0 --binary clang-format-6.0 "${TRAVIS_BRANCH}"
```

Comment:
:question: What's the point of both specifying `git-clang-format-6.0` and `clang-format-6.0`?

Reply:
Without the `--binary` option the command tries to use `clang-format` as a binary, which does not exist. Hence the name of the binary needs to be specified (which just happens to be almost the same, except for the `git-` prefix).